tests/test_report_writing.py

```diff
@@ -38,3 +40,27 @@ def test_school_name_replaced_in_paragraphs(data_path, templates_dir, temp_out_d
             output_paragraphs.append(paragraph.text)
 
     assert "School A" not in output_paragraphs[0]
+
+
+def test_school_table_filters_applied(data_path: Path, templates_dir: Path):
+    """
+    Given a school that has data that is not within the date range
+    When the template is populated with redcap data
+    Then this template should be filled with the following:
+    Table 1, 2, 5: test_1_2021-22, test_3_2021-22, test_4_2021-22, test_5_2021-22, test_6_2021-22
+    Table 3&4: test_1_2021-22, test_4_2021-22
+    Table 6: No data
+    """
+
+    nested_data = parse_masterfile(data_path / "example_masterfile.xlsx")
+    testing_df = join_masterfile_dfs(nested_data)
+    test_pupils_school_data = school_filter(testing_df, "RRS2030220")
+
+    # assert test_pupils_school_data [(test_pupils_school_data.reg_rr_title != "Teacher Leader") |
```

Comment:
Might be worth having a look at our other tests for how we assert specific values, but also  tap @harryjmoss if you're blocked on it